app/routes/goal_routes.py

```diff
@@ -1 +1,89 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, abort, make_response, request, Response
+from dotenv import load_dotenv
+from app.models.goal import Goal
+from app.models.task import Task
+from .route_utilities import validate_model, create_model_from_dict
+import os
+import requests
+from ..db import db
+load_dotenv()
```

Comment:
You have this function call in your `__init__.py` so you can remove it here being that it only needs to be called once. Plus, it's best to have it in your application entry point, `__init__.py`.
```suggestion
```